certstrap/tasks/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -o errexit -o nounset -o xtrace
+
+export GOPATH=$PWD/go
+
+package="github.com/square/certstrap"
+go build -ldflags='-s -w' "${package}"
```

Comment:
Is this really necessary? How much do we save with this? What if it explodes one day?